security/pkg/pki/ca/selfsignedcarootcertrotator.go

```diff
@@ -64,11 +67,24 @@ func NewSelfSignedCARootCertRotator(config *SelfSignedCARootCertRotatorConfig,
 		config:              config,
 		ca:                  ca,
 	}
+	if config.enableJitter {
+		// Select a back off time in seconds, which is in the range of [0, rotator.config.CheckInterval).
+		backOffSeconds := int(time.Duration(rand.Int63n(int64(rotator.config.CheckInterval))).Seconds())
+		rotator.backOffTime = time.Duration(backOffSeconds) * time.Second
+		rootCertRotatorLog.Infof("Set up back off time %s to start rotator.", rotator.backOffTime.String())
+	} else {
+		rotator.backOffTime = time.Duration(0)
+	}
 	return rotator
 }
 
 // Run refreshes root certs and updates config map accordingly.
 func (rotator *SelfSignedCARootCertRotator) Run(rootCertRotatorChan chan struct{}) {
+	if rotator.config.enableJitter && rotator.backOffTime > time.Duration(0) {
```

Comment:
no need to check both variables? since the backoff time is ganranteed to be non zero in line 72?